modules/wmco-prerequisites.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-container-workloads.adoc
+
+[id="wmco-prerequisites_{context}"]
+= Windows Machine Config Operator (WMCO) prerequisites
+
+The following information details the supported cloud provider versions, Windows Server versions, and networking configurations. See the vSphere documentation for any information that is relevant to only that platform.
+
+== Supported Cloud Providers based on OCP and WMCO versions
+
+[cols="3",options="header"]
+|===
+| Cloud Provider           | Supported OKD/OCP Version   | Supported WMCO version
+|Amazon Web Services (AWS) |4.6+ |WMCO 1.0+
+|Azure |4.6+ |WMCO 1.0+
+|VMware vSphere |4.7+ |WMCO 2.0+
+|===
+
+## Supported Windows Server versions
+
+The following table lists the supported link:https://docs.microsoft.com/en-us/windows/release-health/release-information[Windows Server version] based on the applicable cloud provider.
+
+[NOTE]
+====
+Any unlisted Windows Server version are NOT supported and will cause errors. To prevent these errors, only use the appropriate version according to the cloud provider in use.
+====
+
+[cols="2",options="header"]
+|===
+| Cloud Provider      | Supported Windows Server version
+| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809
+| Azure               | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809
+| VMware vSphere      | Windows Server Semi-Annual Channel (SAC): Windows Server 2004
+|===
+
+== Supported Networking
+OVNKubernetes hybrid networking is the only supported networking configuration. See the additional resources below for more information on this functionality. The following tables outline the type of networking configuration and Windows Server versions to be used based on your cloud provider. The network configuration must be completed during the installation of the cluster.
+
+[NOTE]
+====
+OpenShiftSDN networking is the default network for OpenShift clusters. It is NOT supported by WMCO.
+====
+
+.Cloud provider networking support
+[cols="2",options="header"]
+|===
+| Cloud Provider
+| Supported networking
+
+| AWS
+| Hybrid OVNKubernetes
+
+| Azure
+| Hybrid OVNKubernetes
+
+| VMware vSphere
+| Hybrid OVNKubernetes with a Custom VXLAN port
+
+|===
+
+.Hybrid OVNKubernetes Windows Server support
+[cols="2",options="header"]
+|===
+| Hybrid OVNKubernetes  | Supported Windows Server version
+| Default VXLAN port    | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809
+| Custom VXLAN port     | Windows Server Semi-Annual Channel (SAC): Windows Server 2004
+|===
+
+== Supported installation method
+Installer-Provisioned Infrastructure installation method is the only supported installation method. This is
+consistent across all supported cloud providers.
+
+[IMPORTANT]
+====
```

Comment:
I'll repeat my admonishment that you should resist adding admonishments.  The preceding sentence was crystal clear, this one is perfectly clear too.  Actually, the admonishment is adding visual excitement to "what not to do".

Unless you are uncomfortable, lower case "not".  Customers will read it or they won't.--Upper case or lower case won't matter.

The P and I in UPI can be lowercase, not a proper noun.